### PR TITLE
Spider directive is now in the mind

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

Puts the spider directive into the mind.

## Why It's Good For The Game

Spiders can now recall their objective whenever they want.

## Changelog
:cl:
tweak: The directive of sentient spiders is now in the mind.
/:cl:

### CODE REVIEW

**Comment by knacker48** [comment]
You should also give it rainbow text and that wavy effect you used to see on old websites.
image


There's no way they'll miss that

**Comment by IndieanaJones** [comment]
Try as you may, the real problem is greens born from the original changing the directive, making their newborns lack it or have something completely different.  The solution to this is that it be made so any spiders born from the eggs cannot edit their objective, and will pass it onto their offspring.

**Comment by Teepeepee** [comment]
or logging when a directive is set
setting a directive different from your mother's is bad, doing it to self-antag is even worse

**Comment by nemvar** [comment]
I already done both those changes in #43864

File: code/modules/mob/living/simple_animal/hostile/giant_spider.dm [hunk 1]
```diff
@@ -73,7 +73,7 @@
 	..()
 	if(directive)
 		to_chat(src, "<span class='notice'>Your mother left you a directive! Follow it at all costs.</span>")
-		to_chat(src, "<span class='spider'><b>[directive]</b></span>")
+		to_chat(src, "<span class='userdanger'><b>[directive]</b></span>")
```
Comment (ShizCalev) [inline code/modules/mob/living/simple_animal/hostile/giant_spider.dm]: add <big></big> tags instead.

**Comment by 4dplanner** [comment]
The solution I was planning on coding also involved a way to recall the directive, you might want to put that in too

**Comment by nemvar** [comment]
The real fix would be to put the directive into the mind.

**Comment by nemvar** [comment]
Alright. PR now does something completely different but it's better in the long term.

**Comment by ATH1909** [comment]
> Alright. PR now does something completely different but it's better in the long term.

Could you still make it show up in B I G   T E X T ?

**Comment by Repukan** [comment]
Directives are stored in the balls

**Comment by kriskog** [comment]
> 
> 
> I already done both those changes in #43864

I know for a fact there are ways around these changes. I was the original green one day, and  got an order from one of my babies.

**Comment by nemvar** [comment]
I'm not sure you know what you are talking about. Even before my changes, directives could never be passed onto the mother from the child. What you are talking about it probably a midwife, in which case disregard that 'order' because it's effectively just a guy saying something over comms.

Even the best code can't fix retardation.

**Comment by YakumoChen** [comment]
Make it big text so that baby players can't miss it

**Comment by Militaires** [comment]
directives are gay try this instead

**Comment by Arathianen** [comment]
Thanks nemvar, you da real MVP!

**Pull request closed without merging**